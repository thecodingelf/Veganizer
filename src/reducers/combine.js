import { combineReducers } from 'redux';
import authReducer from './AuthReducer'
import userReducer from './UserReducer';

const reducers = combineReducers({
    auth: authReducer,
    user: userReducer,
})

export default reducers