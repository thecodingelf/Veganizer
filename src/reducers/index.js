import { combineReducers } from 'redux';
import AuthReducer from './authentication/AuthReducer';
import UserReducer from './user/UserReducer';

export default combineReducers({
    auth: AuthReducer,
    user: UserReducer,
})