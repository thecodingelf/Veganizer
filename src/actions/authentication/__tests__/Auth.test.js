import { createStore, applyMiddleware } from 'redux';
import ReduxThunk from 'redux-thunk';
import * as actions from '../Auth';
import reducers from '../../../reducers';

const store = createStore(reducers, {}, applyMiddleware(ReduxThunk));

// Testing email changed function.
it('EMAIL CHANGED', () => {
    store.dispatch(actions.emailChanged('user@gmail.com'));
    expect(store).toMatchSnapshot();
});

it('EMAIL CHANGED', () => {
    store.dispatch(actions.emailChanged(null));
    expect(store).toMatchSnapshot();
});