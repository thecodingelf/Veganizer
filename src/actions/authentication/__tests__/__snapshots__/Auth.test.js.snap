// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EMAIL CHANGED 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;

exports[`EMAIL CHANGED 2`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
